WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/UiModelMapper.kt

```diff
@@ -76,6 +78,48 @@ class UiModelMapper
         return mapStatsWithOverview(TimeStatsType.OVERVIEW, useCaseModels, showError)
     }
 
+    @Suppress("CyclomaticComplexMethod")
+    fun mapSubscribers(useCaseModels: List<UseCaseModel>, showError: (Int) -> Unit): UiModel {
```

Comment:
I consolidated (fd93f164acf52e6f3770e15eb9ac14b76e62a7db) some parts of it, there are slight differences in other parts regarding handling nullability. I don't want to make further changes to avoid affecting the production.